Library/Homebrew/cmd/update.rb

```diff
@@ -115,6 +163,12 @@ def load_tap_migrations
   rescue LoadError
     false
   end
+
+  def load_formula_renames
+    load 'formula_renames.rb'
```

Comment:
Ok, in that case, you can change the `require` in `load_tap_migrations` as `load`.